message.module

```diff
@@ -37,13 +37,12 @@ function message_entity_delete(EntityInterface $entity) {
   // Keyed by message ID, pointing to array of the relevant field names.
   $check_mids = [];
 
-  $fields_ids = \Drupal::entityQuery('field_storage_config')->execute();
-  /** @var FieldStorageConfig[] $fields */
-  $fields = entity_load_multiple('field_storage_config', array_keys($fields_ids));
+  /** @var \Drupal\Core\Field\FieldStorageDefinitionInterface[] $fields */
+  $fields = \Drupal::entityTypeManager()->getStorage('field_storage_config')->loadMultiple();
```

Comment:
This doesn't change anything functionally, it's just a little cleaner and removes the deprecated function.